labs/shipped-multi-tier/3.md

```diff
@@ -0,0 +1,98 @@
+## Create an Environment for Deployment
+
+
+1. Click <b>New Environment</b>.
+1. Enter the name for the environment, Test for example.
+1. Click **Save**. 
+
+    ![](posts/files/shipped-multi-tier/assets/4.png)
+
+You are now ready to deploy *Postgres*, *vote-api*, and *vote-ui* services. The order of service deployment is important. Because Postgres has a set of default environment variables that other two services depend on, therefore it is the first to be deployed. After Postgres is deployed, you can copy those variables and apply them to vote-api and vote-ui. 
+
+### Deploy Postgres
+
+
+1. Click the **Deployment** tab.
+
+    ![](posts/files/shipped-multi-tier/assets/deploy-postgres.png)
+
+1. Select **Postgres** and the environment (Test), click **Deploy Build**. 
+
+1. Copy the environment variables of Postgress, which you need to set for vote-api and vote-ui. 
+
+    2. Click **Settings**. 
+    3. Scroll down to the **Deployment Configurations** section. 
+    4. Copy the environment variables.
+
+        ![](posts/files/shipped-multi-tier/assets/postgres-settings.png)
+
+
+### Deploy vote-api 
+
+1. Click **Back to Environments**. 
+
+    ![](posts/files/shipped-multi-tier/assets/vote-api.png)
+
+1. Click the vote-api icon, then **Settings**.
+
+3. Scroll down to the **Deployment Configurations** section. 
+4. Click **Add New variable**, and enter the key value pair. 
+
+    In addition to the variables copied from Postgres, you also need to set the API endpoint so the UI application will read the same variables. 
+
+    **Note**: Do not change the key value.
+
+    The following example shows the vote-api variables. 
+
+      ![](posts/files/shipped-multi-tier/assets/vote-api-cfg.png)
+
+
+<table><tr>
+<td>Key</td>
+<td>Value</td>
+</tr>
+<tr>
+<td>SEED_PORT</td>
+<td>4001</td>
+</tr>
+<tr>
+<td>HOST_VOTE_API</td>
+<td>http://test--multitierapp--vote-api--254b61.gce.shipped-cisco.com</td>
+</tr>
+<tr>
+<td>HOST_VOTE_UI</td>
+<td>http://test--multitierapp--vote-ui--247fa6.gce.shipped-cisco.com</td>
+</tr>
+<tr>
+<td>SEED_HOST</td>
+<td>test--multitierapp--postgres--d1329f-0.service.consul</td>
+</tr>
+<tr>
+<td>HOST_POSTGRES</td>

```

Comment:
We just need to copy single environment variable. HOST_POSTGRES

Once Dinesh fix is pushed, avoid asking user to copy SEED_HOST, PORT etc.